.github/workflows/verify-plugins-installation.yml

```diff
@@ -0,0 +1,67 @@
+name: Check Knative Plugin Installation
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
I'd suggest a common trigger combination that we use in other Knative GH actions. 

```
on:
  push:
    branches: [ 'main', 'release-*' ]

  pull_request:
    branches: [ 'main', 'release-*' ]
```

Reply:
Ah, I will change it accordingly.